packages/react-scripts/scripts/build.js

```diff
@@ -28,6 +28,7 @@ var webpack = require('webpack');
 var config = require('../config/webpack.config.prod');
 var paths = require('../config/paths');
 var checkRequiredFiles = require('react-dev-utils/checkRequiredFiles');
+var fileSizeReporter = require('react-dev-utils/fileSizeReporter');
 var recursive = require('recursive-readdir');
 var stripAnsi = require('strip-ansi');
 
```

Comment:
Please add

```js
var measureFileSizesBeforeBuild = FileSizeReporter.measureFileSizesBeforeBuild;
var printFileSizesAfterBuild = FileSizeReporter.printFileSizesAfterBuild;
```

here instead of referencing them with `FileSizeReporter` later.